tests/nightly/apache_rat_license_check/rat-excludes

```diff
@@ -57,3 +57,4 @@ apache-rat-tasks/*
 moderngpu/*
 deformable_im2col.cuh
 deformable_im2col.h
+ci/qemu/.*
```

Comment:
Why?

Reply:
Ok, gotcha

Reply:
Are we sure we can exclude everything in there? We have to make sure that the os specific files comply with the Apache license or otherwise we will get into legal trouble. Could you elaborate?

Reply:
@marcoabreu if you have an OS specific config file like inittab, does it need to have a license as well?

Reply:
Yes, everything we publish in the repository needs a licence